Params not running properly

When I run the Embroidery extension:
Embroider..  works
Params gives me the following error:

Traceback (most recent call last):
  File "embroider_params.py", line 618, in <module>
    e.affect()
  File "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex.py", line 283, in affect
    self.effect()
  File "embroider_params.py", line 592, in effect
    frame = SettingsFrame(tabs_factory=self.create_tabs)
  File "embroider_params.py", line 316, in __init__
    self.preset_chooser = wx.ComboBox(self, wx.ID_ANY, style=wx.CB_SORT)
wxAssertionError: C++ assertion "!(style & wxCB_SORT)" failed at /Users/robind/projects/buildbots/macosx-vm6/dist-osx-py27/Phoenix/ext/wxWidgets/src/osx/combobox_osx.cpp(60) in Create(): wxCB_SORT not currently supported by wxOSX/Cocoa

I modified embroider_params.py and now it will run without crashing inkscape  and gives the message below.  I do not get the tabbed box shown in your 12/6/17 **very nice** "how to" embroider the inkscape logo.  That was helpful to see what it should look like.  

I bet I need to adjust my install in some way 
2011 Mac Air OS 
"High Sierra 10.13.1;
Inkscape 0.92.2 5c3e80d, 2017-08-06;
X11 Xquartz 2.7.7   


Any thoughts on what to try or where to look?  Thanks very much for your help and for the program!


Error message:
Traceback (most recent call last):
  File "embroider_params.py", line 618, in <module>
    e.affect()
  File "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex.py", line 283, in affect
    self.effect()
  File "embroider_params.py", line 592, in effect
    frame = SettingsFrame(tabs_factory=self.create_tabs)
  File "embroider_params.py", line 316, in __init__
    self.preset_chooser = wx.ComboBox(self, wx.ID_ANY, style=wx.CB_SORT)
wxAssertionError: C++ assertion "!(style & wxCB_SORT)" failed at /Users/robind/projects/buildbots/macosx-vm6/dist-osx-py27/Phoenix/ext/wxWidgets/src/osx/combobox_osx.cpp(60) in Create(): wxCB_SORT not currently supported by wxOSX/Cocoa

--- a/embroider_params.py
+++ b/embroider_params.py
@@ -281,7 +281,7 @@ class ParamsTab(ScrolledPanel):
 
                 input.Bind(wx.EVT_CHECKBOX, self.changed)
             elif len(param.values) > 1:
-                input = wx.ComboBox(self, wx.ID_ANY, choices=param.values, style=wx.CB_DROPDOWN | wx.CB_SORT)
+                input = wx.ComboBox(self, wx.ID_ANY, choices=sorted(param.values), style=wx.CB_DROPDOWN)
                 input.Bind(wx.EVT_COMBOBOX, self.changed)
                 input.Bind(wx.EVT_TEXT, self.changed)
             else:
@@ -313,7 +313,7 @@ class SettingsFrame(wx.Frame):
 
         self.presets_box = wx.StaticBox(self, wx.ID_ANY, label="Presets")
 
-        self.preset_chooser = wx.ComboBox(self, wx.ID_ANY, style=wx.CB_SORT)
+        self.preset_chooser = wx.ComboBox(self, wx.ID_ANY)
         self.update_preset_list()
 
         self.load_preset_button = wx.Button(self, wx.ID_ANY, "Load")
@@ -344,7 +344,7 @@ class SettingsFrame(wx.Frame):
     def update_preset_list(self):
         preset_names = load_presets().keys()
         preset_names = [preset for preset in preset_names if preset != "__LAST__"]
-        self.preset_chooser.SetItems(preset_names)
+        self.preset_chooser.SetItems(sorted(preset_names))
 
     def get_preset_name(self):
         preset_name = self.preset_chooser.GetValue().strip()

